src/components/svg/BlobbyGrayIconImage.tsx

```diff
@@ -11,12 +11,12 @@ const SvgBlobbyGrayIconImage = ({ title, titleId, ...props }: SVGProps<SVGSVGEle
       <path
         fill='#F1F0EC'
         fillRule='evenodd'
-        d='M113.962 6.222c26.535 2.074 38.447 30.862 44.693 56.735 4.621 19.146-2.901 38.537-18.932 49.978-15.595 11.131-35.55 10.486-51.986.639C63.731 99.19 34.311 80.943 40.953 53.757c7.619-31.19 41-50.037 73.009-47.535Z'
```

Comment:
Due to an update of dependencies many existing svg based assets were updated to remove the closepath Z/z.  This doesn't affect the rendering of the assets.